html/home-page.html

```diff
@@ -0,0 +1,157 @@
+                </div>
+                <ul class="nav-menu">
+                    <li class="nav-list"><a href="../html/home-page.html" class="nav-link">Home</a></li>
+                    <li class="nav-list"><a href="../html/about-page.html" class="nav-link">About</a></li>
+                    <li class="nav-list"><a href="#" class="nav-link">Program</a></li>
+                    <li class="nav-list"><a href="#" class="nav-link">Join</a></li>
+                    <li class="nav-list"><a href="#" class="nav-link">News</a></li>
+                    <li class="campaign nav-list"><a href="#" class="nav-link">CC Campaign</a></li> 
+                </ul>
+            </div>
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+        </div>
+    </header>
+    <main>
+        
+        <section class="container-fluid head">
+            <div class="container headline">
+            <h1 class="greetings">Hello! World</h1>
+        <h1 class="title"> Let's built  from <br>here</h1>
+        <p class="narates"> GitHub Universe 2022 is in the books but you’re still able to view sessions from all our content tracks. If you registered for the event you can login below, or sign-up as a first time viewer for our session catalog.</p>
+        <h2 class="date">2022.11.09(WED)~10(THU)</h2>
+        <h3 class="venu">Virtual Schedule @watch.githubuniverse.com</h3>
+            </div>
+            <img src="../images/Hero_Gradient.webp" alt="">
+        </section>
+
+        <section class="container-fluid pg ">
+        <div class="main">
+        <h3 class="main-title">Main Program</h3>
+        
+        <span class="line"></span>
+        
+        <div class="container Program-content">
+                <ul>
+                    <li>
+                        <img src="../images/asterisk_0.svg" alt="cloud pic">
+                        <a href="#">Cloud</a>
+                        <p>Learn how the best companies, developers, and start-ups are using software in the cloud to accelerate productivity and transform their business. </p>
+                    </li>
+                    <li>
+                        <img src="../images/asterisk_1.svg" alt="security pic">
+                        <a href="#">Security</a>
+                        <p>Hone your security skills by exploring the GitHub tools, techniques, and best practices for securing your software development lifecycle. 
+                        </p>
+                    </li>
+                    <li>
+                        <img src="../images/asterisk_2.svg" alt="AI pic">
+                        <a href="#">AI</a>
+                        <p>See how GitHub, businesses, and the machine learning and AI community build exciting solutions to address current software challenges. </p>
+                    </li>
+                </ul>
+                
+         </div>
```

Comment:
- [ ] Great job adding the articles 💯, but please align the logos and the description of each article in the mobile version. It is to stick as much as possible with the original template. 🏒
- [ ] Kindly, add 2 extra articles, in the template appear 5 articles. 📦
- [ ] Kindly, use `<article>` tag, it is semantically correct, and perfect for this section because you are creating articles 🖌️ .

Your design:

![image](https://user-images.githubusercontent.com/91301423/203668153-230a36d9-4615-4ebd-afa1-19f805cc5a50.png)

Original design:

![image](https://user-images.githubusercontent.com/91301423/203668205-baed267b-1643-4200-a786-e25253f4288b.png)